library/core/src/task/wake.rs

```diff
@@ -71,6 +71,12 @@ impl RawWaker {
 /// pointer of a properly constructed [`RawWaker`] object from inside the
 /// [`RawWaker`] implementation. Calling one of the contained functions using
 /// any other `data` pointer will cause undefined behavior.
+///
+/// These functions must all be thread-safe (even though [`RawWaker`] is
+/// <code>\![Send] + \![Sync]</code>)
```

Comment:
```suggestion
/// `\![Send] + \![Sync]`)
```
I think the escapes can be deleted as well, though I'm not sure.

Reply:
links don't work in ``` ``` ``s.

Reply:
Ah, then it makes sense.

Reply:
And the `\!` escapes too are needed to prevent interpretation as an image embed.